packages/core/src/utils/Utils.ts

```diff
@@ -822,6 +822,20 @@ export class Utils {
     return !!GroupOperator[key];
   }
 
+  static hasNestedKey(object: any, key: string): boolean {
```

Comment:
let's not use `any`, either its object, so `Dictionary`, or `unknown`
```suggestion
  static hasNestedKey(object: Dictionary, key: string): boolean {
```